README.md

```diff
@@ -2,31 +2,31 @@
 
 ## Overview
 
-The current Gaia Version of the Cosmos Hub mainnet is [`v10.0.x`](https://github.com/cosmos/gaia/releases/tag/v10.0.1). To bootstrap a mainnet node, it is possible to sync from `v9.1.1` via Quicksync or via [State Sync](https://hub.cosmos.network/main/hub-tutorials/join-mainnet.html#state-sync).
+The current Gaia Version of the Cosmos Hub mainnet is [`v11.0.0`](https://github.com/cosmos/gaia/releases/tag/v11.0.0). To bootstrap a mainnet node, it is possible to sync from `v10.0.2` via Quicksync or via [State Sync](https://hub.cosmos.network/main/hub-tutorials/join-mainnet.html#state-sync).
```

Comment:
if we give specific v11.0.0 we have to update this doc each time we do minor releases ?

Reply:
Will add a documentation variable in the near future, so that it will update all documents.